README.md

```diff
@@ -181,6 +182,34 @@ const b = asUnit('1rem') // returns '1rem'
 const c = asUnit(null) // Throws a TypeError
 ```
 
+`asMaybe` creates a cleaner that doesn't throw on an invalid type. It tries the cleaner, and if that throws an exception, it will return undefined instead:
+
+```typescript
+// Makes a Cleaner<string | undefined>:
+const asMaybeString = asMaybe(asString)
+
+const a = asUnit('Valid string') // returns 'Valid string'
+const b = asUnit(23) // returns undefined
+const c = asUnit(null) // returns undefined
+```
+
+This cleaner is useful as a type guard on your data:
+
+```typescript
+const pizza = asMaybe(asPizza)(obj)
+const salad = asMaybe(asSalad)(obj)
+
+if (pizza != null) {
+  // It's a pizza
+} else if (salad != null) {
+  // It's a salad
+} else {
+  // It's neither
+}
+```
+
+**Caution**: This type will silence all exceptions from the cleaner(s) it composes. Only use on types for which you do not care why a value is not valid.
```

Comment:
The big ***Caution*** seems a bit much. This isn't really dangerous; it's just part of the way it works.

Reply:
You're right, though I feel it's worth mentioning for users as it could be a source of bugs if used incorrectly, no?

Reply:
Right, the rest of the sentence should stay, since it's useful. Just the "Caution" word is scary.

Reply:
That's understandable. I'll remove the "caution" word.